spring-cloud-kubernetes-examples/kubernetes-leader-election-example/pom.xml

```diff
@@ -78,6 +78,14 @@
 					<skip>true</skip>
 				</configuration>
 			</plugin>
+			<plugin>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<version>2.22.2</version>
```

Comment:
This should happen in spring-cloud-build

Reply:
I see that currently these version is taken from parent pom.xml (branch master). I changed both versions for maven-failsafe-plugin and maven-surefire-plugin in the parent pom.xml.

		<plugins>
			<plugin>
				<groupId>org.apache.maven.plugins</groupId>
				<artifactId>maven-failsafe-plugin</artifactId>
				<version>${maven-failsafe-plugin.version}</version>
			</plugin>
		</plugins>

I don't know - maybe it should be taken from spring-cloud-build, but I'm guessing that it is the change for the whole Spring Cloud project. I had to set version of maven-surefire-plugin to 2.22.2 to make test works after changing enabling junit-jupiter for spring-cloud-kubernetes-leader module.